github/respository_rules_utils.go

```diff
@@ -472,6 +472,29 @@ func flattenRules(rules []*github.RepositoryRule, org bool) []interface{} {
 			rule["required_check"] = requiredStatusChecksSlice
 			rule["strict_required_status_checks_policy"] = params.StrictRequiredStatusChecksPolicy
 			rulesMap[v.Type] = []map[string]interface{}{rule}
+
+		case "workflows":
+			var params github.RequiredWorkflowsRuleParameters
+			log.Printf("[DEBUG] osama test: %v", v.Parameters)
+
+			err := json.Unmarshal(*v.Parameters, &params)
+			if err != nil {
+				log.Printf("[INFO] Unexpected error unmarshalling rule %s with parameters: %v",
+					v.Type, v.Parameters)
+			}
+
+			requiredWorkflowsSlice := make([]map[string]interface{}, 0)
+			for _, workflow := range params.RequiredWorkflows {
+				requiredWorkflowsSlice = append(requiredWorkflowsSlice, map[string]interface{}{
+					"repository_id": workflow.RepositoryID,
+					"ref":           *workflow.Ref,
+					"path":          workflow.Path,
+				})
+			}
+
+			rule := make(map[string]interface{})
+			rule["required_workflow"] = requiredWorkflowsSlice
+			rulesMap[v.Type] = []map[string]interface{}{rule}
```

Comment:
```suggestion
			rulesMap["required_workflows"] = []map[string]interface{}{rule}
```

you cannot use "v.Type" because it is named [required_workflows](https://github.com/integrations/terraform-provider-github/blob/main/github/resource_github_organization_ruleset.go#L424) in the scheme, not **workflows**. Look in my [PR](https://github.com/integrations/terraform-provider-github/pull/2488), I also add "do_not_enforce_on_create" to the scheme